module/VuFind/tests/unit-tests/src/VuFindTest/Session/DatabaseTest.php

```diff
@@ -1,150 +0,0 @@
-<?php
-
-/**
- * Database Session Handler Test Class
- *
- * PHP version 8
- *
- * Copyright (C) Villanova University 2019.
- *
- * This program is free software; you can redistribute it and/or modify
- * it under the terms of the GNU General Public License version 2,
- * as published by the Free Software Foundation.
- *
- * This program is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU General Public License for more details.
- *
- * You should have received a copy of the GNU General Public License
- * along with this program; if not, write to the Free Software
- * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
- *
- * @category VuFind
- * @package  Tests
- * @author   Demian Katz <demian.katz@villanova.edu>
- * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
- * @link     https://vufind.org/wiki/development:testing:unit_tests Wiki
- */
-
-namespace VuFindTest\Session;
-
-use VuFind\Session\Database;
-
-/**
- * Database Session Handler Test Class
- *
- * @category VuFind
- * @package  Tests
- * @author   Demian Katz <demian.katz@villanova.edu>
- * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
- * @link     https://vufind.org/wiki/development:testing:unit_tests Wiki
- */
-class DatabaseTest extends \VuFindTest\Unit\SessionHandlerTestCase
```

Comment:
I don't think we should completely delete this test -- it's still useful to have a test that covers \VuFind\Session\Database and confirms that it proxies the appropriate methods of the database service, just as this test did for the table class. Rather than deleting this, I think it would be better to update it. Probably a matter of adding service support to SessionHandlerTestCase and then swapping out $this->tables for $this->services in appropriate places. Let me know if you need more help!